aten/src/ATen/native/ConvolutionMM2d.cpp

```diff
@@ -410,34 +417,44 @@ std::tuple<Tensor&, Tensor&> slow_conv2d_forward_out_cpu(
                   n_input_plane * kernel_height * kernel_width,
                   output_height * output_width});
   }
-  output.resize_({batch_size, n_output_plane, output_height, output_width});
 
-  at::parallel_for(0, batch_size, 0, [&](int64_t start, int64_t end) {
-    NoGradGuard no_grad;
-    AutoDispatchBelowADInplaceOrView non_variable_type_mode;
-    for (int64_t t = start; t < end; t++) {
-      Tensor input_t = input[t].unsqueeze(0);
-      Tensor output_t = output[t];
-      Tensor finput_t = finput[t];
-      slow_conv2d_update_output_frame(
-          input_t,
-          output_t,
-          weight_2d,
-          bias,
-          finput_t,
-          kernel_height,
-          kernel_width,
-          stride_height,
-          stride_width,
-          pad_height,
-          pad_width,
-          n_input_plane,
-          input_height,
-          input_width,
-          n_output_plane,
-          output_height,
-          output_width);
-    }
+  output.resize_({batch_size, n_output_plane, output_height, output_width});
+  if (bias.defined()) {
+    output.copy_(bias.reshape({-1, 1, 1}));
+  }
+  TORCH_CHECK(output.is_contiguous());
```

Comment:
can you also please check that `finput` is contiguous? It wasn't done before, but `finput` can be discontiguous if it is passed as a discontiguous tensor of the correct size.